modules/fedimint-ln-client/src/pay.rs

```diff
@@ -227,6 +261,8 @@ pub enum GatewayPayError {
     },
     #[error("OutgoingContract was not created in the federation")]
     OutgoingContractError,
+    #[error("Gateway did not respond in time")]
+    Timeout,
```

Comment:
Is this being used?

Reply:
Good catch! I introduced this error variant to handle the "API call took too long but didn't return error" case that you rightfully argued against being an error case.